src/schema/v2/conversation/update_conversation_mutation.ts

```diff
@@ -1,34 +1,66 @@
+import { GraphQLBoolean } from "graphql"
 import { GraphQLString, GraphQLNonNull } from "graphql"
 import { mutationWithClientMutationId } from "graphql-relay"
 import Conversation from "schema/v2/conversation"
 import { ResolverContext } from "types/graphql"
 
-export default mutationWithClientMutationId<any, any, ResolverContext>({
+interface UpdateMessageMutationInputProps {
+  conversationId: string
+  fromLastViewedMessageId: string
+}
+
+export default mutationWithClientMutationId<
+  UpdateMessageMutationInputProps,
+  any,
+  ResolverContext
+>({
   name: "UpdateConversationMutation",
   description: "Update a conversation.",
   inputFields: {
     conversationId: {
       type: new GraphQLNonNull(GraphQLString),
       description: "The id of the conversation to be updated.",
     },
+    dismissed: {
+      type: GraphQLBoolean,
+      description: "Mark the conversation as dismissed",
+    },
     fromLastViewedMessageId: {
-      type: new GraphQLNonNull(GraphQLString),
+      type: GraphQLString,
       description: "The message id to mark as read.",
     },
+    sellerOutcome: {
+      type: GraphQLString,
+      description:
+        "The seller outcome for the conversation. Options include `already_contacted`, `dont_trust`, `other`, `work_unavailable`.",
+    },
+    sellerOutcomeComment: {
+      type: GraphQLString,
+      description: "The seller outcome comment for the conversation.",
+    },
   },
   outputFields: {
     conversation: {
       type: Conversation.type,
       resolve: (conversation) => conversation,
     },
   },
-  mutateAndGetPayload: (
-    { conversationId, fromLastViewedMessageId },
+  mutateAndGetPayload: async (
+    { conversationId, ...args },
     { conversationUpdateLoader }
   ) => {
-    if (!conversationUpdateLoader) return null
-    return conversationUpdateLoader(conversationId, {
-      from_last_viewed_message_id: fromLastViewedMessageId,
-    })
+    if (!conversationUpdateLoader) {
+      return null
+    }
+
+    try {
+      const updatedConversation = await conversationUpdateLoader(
+        conversationId,
+        args
```

Comment:
I think this might  be a real bug, you're passing in `args` directly, but they are camel-cased, and this loader and upstream API requires them to be snake-cased.